src/Phantoman.php

```diff
@@ -217,7 +217,10 @@ private function getCommandParameters()
      */
     private function getCommand()
     {
-        return escapeshellarg(realpath($this->config['path'])) . ' ' . $this->getCommandParameters();
+        // Prefix command with exec on non Windows systems to ensure that we receive the correct pid.
+        // See http://php.net/manual/en/function.proc-get-status.php#93382
+        $commandPrefix = $this->isWindows() ? '' : 'exec ';
+        return escapeshellarg($commandPrefix . realpath($this->config['path'])) . ' ' . $this->getCommandParameters();
```

Comment:
Thanks for the PR!

Currently, PhantomJS is failing to start as `exec` is getting added within `escapeshellarg`. When you have some time, can you change this line to read

```php
return $commandPrefix . escapeshellarg(realpath($this->config['path'])) . ' ' . $this->getCommandParameters();
```